### PR TITLE
First Pull request for addition of Landing Page

### PR DESCRIPTION
**Pull request by Saloni242**
# Description

The time period of development (from fork to pull request): 

Forked on 11th Sept 2020 and made changes on 12th Sept 2020. Added the Landing Page as Home page and then added Particles.js in the background of Landing Page

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/Footer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Grid, Navbar,NavItem,Nav} from 'react-bootstrap';
```
Comment (GreenBayRules) [inline src/Footer.js:2]: Let's keep formatting/whitespace all consistent.

You can do `import {Grid, Navbar, NavItem, Nav} from 'react-bootstrap';` or  `import { Grid, Navbar, NavItem, Nav } from 'react-bootstrap';` but keep it consistent

File: src/Home.js [hunk 2]
```diff
@@ -1,11 +1,30 @@
-import React, { Component } from 'react'; 
+import React, { Component } from 'react';
+import Header from './Header.js';
+import Footer from './Footer.js';
+import './css/Header_Footer.css';
+import {Form,FormControl,Button} from 'react-bootstrap';
```
Comment (GreenBayRules) [inline src/Home.js:5]: same thing here

File: src/Home.js [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+        <Particles  canvasClassName="example"
+        params={{
+      "particles":{"number":{"value":80,"density":{"enable":true,"value_area":800}},"color":{"value":"#62a337"},"shape":{"type":"circle","stroke":{"width":0,"color":"#000000"},"polygon":{"nb_sides":5},"image":{"src":"img/github.svg","width":100,"height":100}},"opacity":{"value":0.5,"random":false,"anim":{"enable":false,"speed":1,"opacity_min":0.1,"sync":false}},"size":{"value":3,"random":true,"anim":{"enable":false,"speed":40,"size_min":0.1,"sync":false}},"line_linked":{"enable":true,"distance":150,"color":"#62a337","opacity":0.4,"width":1},"move":{"enable":true,"speed":6,"direction":"none","random":true,"straight":false,"out_mode":"out","bounce":false,"attract":{"enable":false,"rotateX":600,"rotateY":1200}}},"interactivity":{"detect_on":"canvas","events":{"onhover":{"enable":true,"mode":"repulse"},"onclick":{"enable":true,"mode":"push"},"resize":true},"modes":{"grab":{"distance":400,"line_linked":{"opacity":1}},"bubble":{"distance":400,"size":40,"duration":2,"opacity":8,"speed":3},"repulse":{"distance":200,"duration":0.4},"push":{"particles_nb":4},"remove":{"particles_nb":2}}},"retina_detect":true
+      }} />
```
Comment (GreenBayRules) [inline src/Home.js:17]: Can you indent lines 15-17?

**Review by GreenBayRules — CHANGES_REQUESTED** [review]
nice job! please fix these formatting changes and we can merge

**Comment by GreenBayRules** [comment]
@Saloni242 Any update on this?

**Comment by Saloni242** [comment]
New design decided. So we are not working on the previous design.

**Pull request closed without merging**